regexes/device/mobiles.yml

```diff
@@ -5019,7 +5024,7 @@ Casper:
 
 # Cat (Caterpillar, Catphones)
 Cat:
-  regex: 'Cat ?(tablet|stargate|nova)|(?:B15Q|CAT B35|CAT-B100|S62 Pro|S48c)(?:[);/ ]|$)'
+  regex: 'Cat ?(tablet|stargate|nova)|(?:B15Q|CAT B35|CAT-B100|S62 Pro|S48c|Android 9.*; S52|Android 10; S42)(?:[);/ ]|$)'
```

Comment:
after Android <version> there is a language construct
exsample
Android 9; en; Model code) applewebkit/
Android 9; en; Model code build/<os build>) applewebkit/

the  separator ";", present only in parentheses.

as for the Cat S52, it exists only since version 9 of android 
https://www.gsmarena.com/cat_s52-9938.php

The s52 code can also occur for Doogee, on Android version 12, and I don't have such a user agent.

![image](https://user-images.githubusercontent.com/1337066/187421101-ac243c85-5ad0-454e-8df5-dc1d98fe3375.png)

https://www.imei.info/phonedatabase/sfr-altice-s42/
https://www.imei.info/phonedatabase/sfr-altice-s32/

Reply:
Ok. I see. But then you could also use something like `Android 9;( \w+;)? S52`